pages/Transfer.qml

```diff
@@ -435,7 +434,7 @@ Rectangle {
             Label {
                 id: privacyLabel
                 fontSize: 15
-                text: ""
+                text: qsTr("Ring size") + translationManager.emptyString
```

Comment:
Hiding the actual ring size being used is probably not what we want to do, and might be harmful in some cases.
I suggest to rephrase the wording to: `qsTr("Ring size: %1).arg(mixin + 1)`

Reply:
You're right idk how I missed this, thanks